tests/django/test_mocking.py

```diff
@@ -0,0 +1,230 @@
+from collections import defaultdict
+
+import pytest
+
+from schematools.contrib.django.db import create_tables
+from schematools.contrib.django.factories import (
+    schema_model_mockers_factory,
+    schema_models_factory,
+)
+from schematools.contrib.django.faker.create import create_data_for
+from schematools.contrib.django.faker.relate import relate_datasets
+from schematools.utils import to_snake_case
+
+from .conftest import DATABASE_URL
+
+
+@pytest.mark.django_db
+@pytest.mark.skipif(DATABASE_URL is None, reason="DATABASE_URL not set")
+def test_mocking_creates_data(gebieden_schema, gebieden_dataset):
+    """Prove that mocking create data."""
```

Comment:
```suggestion
    """Prove that mocking creates data."""
```

Maybe we should get rid of mandatory docstrings, because this one just repeats the function name.

Reply:
I'm all for it to disable this in flake8. Every developer knows when it makes sense to add a docstring.

Reply:
I added D101 and D102 to the flake8 ignores.